frontend/src/router/index.js

```diff
@@ -35,6 +35,17 @@ const titles = {
 const router = new Router({
   base: baseURL,
   mode: "history",
+  scrollBehavior (to, from, saved) {
+    if (saved) {
+      return new Promise((resolve, reject) => {
+        setTimeout(() => {
+          resolve({saved})
+        }, 50)
```

Comment:
Why timeout is needed here?

Reply:
Dude, just accept the PR already. We've been waiting forever. It's functional.

Reply:
This is not an answer to the question

Reply:
@Tamaarine can you please answer. We are waiting for a long time for this feature.

Reply:
> Dude, just accept the PR already. We've been waiting forever. It's functional.

I agree, just accept the PR, short timeout will not break anything.